pages.pl/linux/apt-file.md

```diff
@@ -0,0 +1,20 @@
+# apt-file
+
+> Wyszukiwanie plików w pakietach apt, łącznie z tymi jeszcze nie zainstalowanymi.
+> Więcej informacji: <https://manpages.debian.org/latest/apt-file/apt-file.1.html>.
+
+- Aktualizacja bazy metadanych:
+
+`sudo apt update`
+
+- Wyszukanie pakietu, który zawiera określony plik lub ścieżkę:
+
+`apt-file {{search|find}} {{część/ścieżki/do/pliku}}`
```

Comment:
Shouldn't this be `part/of/filename`'s Polish equivalent ?

Reply:
TBH I'm planning to change the English version.
Right now in description there is _file_ or _path_, but in example there is _part/of/filename_. And format clearly indicates file path, not file name.